subprojects/libnvme.wrap

```diff
@@ -1,6 +1,6 @@
 [wrap-git]
 url = https://github.com/linux-nvme/libnvme.git
-revision = 8efcec730603ffcc379ba0bf821e4c24ec87715b
+revision = 27ed889d0b1917919763816c97a49bb6e29d4547
```

Comment:
btw, could split the libnvme build dependency into a separate patch and have it first? This makes any backporting way simpler.

Reply:
Created #2616

Reply:
Usually you just add a patch to a PR, no need for an extra PR just to update the dependency.